temoa-uiuc/temoa-uiuc-chws.md

```diff
@@ -0,0 +1,49 @@
+# TEMOA's model of the CHWS
+
+* ExistingCapacity [Done]
+* Efficiency [Done]
+* CostInvest: No Invest Cost because UofI already has it
+* CostFixed: To be added in a future PR
+* CapacityToActivity [Done]
+* CapacityFactorTech [To ask]
+* Storage duration [To ask]
+
+# ExistingCapacity
+
+* 1 ton of chilled water ~ 3.5 kWh [1]
+* COP ~ 5 [2]
+* tons to kg ~ 1000
+* [tons]/(1000 * 5) * 3.5 = [MWe]
+* for example: 52,000 tons yield 36 MW
+* UIUC: 37,500 tons
+* UIUC's total capacity: 6.5e6 millon-gallon ~ 24,600 tons
+* capacity: 37,500 tons
+* time to discharge: 39.4 min
```

Comment:
I think this is the value that goes into storage duration, from looking at examples in ``TemoaProject``. The duration is understood as a fraction of the hours in a year. So this would be quite a small number. (2/3)/8760

Reply:
This 'time to discharge' is the time it takes for the CHWS to use all the cold water in the tank if working at full capacity.
For the Storage Duration, I would consider 12 hours. It charges at night, and during the day (around 12 hours later) it discharges.